### DISCUSSION TITLE
个人alist分享

### DISCUSSION THREAD

**Discussion by JIUYINGSHI** [share121/inter-knot#1589] [其他]
**有关webdav的分享，可以拿我的webdav去看番，个人推荐网易的filmly，操作在软件的文档有写
`jiu-ying.iepose.cn`，端口`5244`**
![image](https://github.com/user-attachments/assets/f1aad52d-4b7d-4559-9562-e4dad2753627)

**事先说明,番剧更新是定时上传网盘的，但是有什么全看我爱看什么了😊**

**Comment by capacitor1** [comment]
## 我嘞个豆

我也开了个alist但可惜没法分享
<details>
<summary></summary>

*分享了会被治安官逮捕的吧doge*

</details>

**Reply by JIUYINGSHI** [reply]
如果违规了的话就删掉了
